### PR TITLE
JENKINS-59016 fixing issues with certificates when using org

### PR DESCRIPTION
**Pull request by joseblas**
# Description

When using an Organisation credentials are not used properly
See 
[JENKINS-59016](https://issues.jenkins-ci.org/browse/JENKINS-59016) for further information. 

# Submitter checklist
- [X] Link to JIRA ticket in description, if appropriate.
- [X] Change is code complete and matches issue description
- [ ] Automated tests have been added to exercise the changes
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verify that the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

# Documentation changes
- [ ] Link to jenkins.io PR, or an explanation for why no doc changes are needed

# Users/aliases to notify

### CODE REVIEW

**Review by car-roll — APPROVED** [review]

**Review by olamy — APPROVED** [review]

File: src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/config.jelly [hunk 1]
```diff
@@ -8,9 +8,9 @@
       </f:entry>
     </j:when>
     <j:otherwise>
-      <f:invisibleEntry>
-        <f:select field="apiUri"/>
-      </f:invisibleEntry>
+      <f:entry title="${%API endpoint}" field="apiUri">
+        <f:textbox readonly="readonly"/>
+      </f:entry>
```
Comment (daniel-beck) [inline src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/config.jelly:13]: This change is really weird. On a pristine instance, I get this uneditable text field in an org folder configuration:

Screenshot 2025-08-29 at 17 12 25

Why this change?
